views/default/lightbox/elgg-colorbox-theme/colorbox.css

```diff
@@ -127,7 +123,7 @@
 }
 
 #cboxLoadingOverlay {
-	background: transparent url(graphics/ajax_loader_bw.gif) no-repeat center center;
+	background: transparent url(../../graphics/ajax_loader_bw.gif) no-repeat center center;
```

Comment:
tested it... this line does not need to be changed the reason the css is not loading on the backend is because it is only extended on the elgg.css and not on the admin.css. Will you change it, or should i provide a new PR?